proxy/src/main/java/org/dragonet/proxy/network/translator/pe/PEInventoryTransactionPacketTranslator.java

```diff
@@ -1,181 +0,0 @@
-package org.dragonet.proxy.network.translator.pe;
```

Comment:
How do you manage to delete a whole file and commit that? Next time, please check all your changes bebore commiting.